src/web/components/Onwards/Onwards.tsx

```diff
@@ -102,6 +106,27 @@ export const Onwards = ({
     } else if (isAdFreeUser && isPaidContent) {
         // Don't show any related content (other than story packages) for
         // adfree users when the content is paid for
+    } else if (seriesTag) {
```

Comment:
blog as well maybe? 
 https://github.com/guardian/frontend/blob/027d9f7ffa5f7382f453c0ccb6f760853e9c714e/static/src/javascripts/bootstraps/enhanced/trail.js#L128

Reply:
Ah, blogs, nice. Shall add a ticket for this. And for Rob's Onwards too. Robwards?

Reply:
Also 'opinion'. May be covered by blog https://docs.google.com/presentation/d/1rfUXsFXQDffnEQX7UhApsw1xUi1s9xAqEJa9twVqpPE/edit#slide=id.g4686b07a52_1_18

Reply:
Yeah, this blog tone (`commentisfree/commentisfree`) override is a stinker. It displays as “DEPRECATED” in TagManager for years and yet is used every day by editors. AFAIK, this is solely to override the default behaviour and display Opinion (instead of a main keyword) at top of this IA thingie here:

![image](https://user-images.githubusercontent.com/6032869/73611410-7fa59800-45d9-11ea-932d-39280a1e9254.png)

Reply:
Ah, that explains a few things. We have [this code](https://github.com/guardian/dotcom-rendering/blob/3d2fa5e3814674916e8398f25ccf175cdf6bde6b/src/web/components/SeriesSectionLink.tsx#L70) but I never really understood how it came about.

To do the same thing in Onwards, I raised [this ticket](https://trello.com/c/FvIOPwdi/1132-blog-series-onwards-content)

Reply:
Actually, in the end the change was so small and seems so relevant to this context that I added blogs here to this PR